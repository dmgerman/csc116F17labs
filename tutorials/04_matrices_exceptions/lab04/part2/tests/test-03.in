4 3
6 10 6
10 6 10
6 10 6
10 6 10

4 3
10 6 10
6 10 6
10 6 10
6 10 6