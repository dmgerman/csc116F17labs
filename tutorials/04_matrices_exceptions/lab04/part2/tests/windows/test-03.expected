Reading first matrix
Enter the number of rows and columns: 
Enter the elements of the matrix: 

Successfully read a matrix with dimensions 4 x 3
M1: 
6 10 6 
10 6 10 
6 10 6 
10 6 10 
Reading second matrix
Enter the number of rows and columns: 
Enter the elements of the matrix: 

Successfully read a matrix with dimensions 4 x 3
M2: 
10 6 10 
6 10 6 
10 6 10 
6 10 6 
Sum of M1 and M2: 
16 16 16 
16 16 16 
16 16 16 
16 16 16 
Error: Unable to compute trace of a non-square matrix
